codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoder.java

```diff
@@ -427,16 +420,16 @@ public void error(Throwable cause) {
 
         @Override
         public boolean write(int allowedBytes) {
```

Comment:
@Scottmitch well we don't throw an Exception in here, so what's the point in adding it to the signature?